addons/base_import/static/src/js/import_action.js

```diff
@@ -515,7 +526,20 @@ var DataImport = AbstractAction.extend({
                 var $commentCell = $fieldInput.closest('tr.oe_import_grid-row').find('.oe_import_comment_cell');
                 var $optionsDiv = $commentCell.find('.oe_import_options_div');
                 var isRelational = fieldInfo.type === 'many2many' || fieldInfo.type === 'many2one';
-                var showSkip = !fieldInfo.required && (isRelational || fieldInfo.type === 'selection');
+                const showSkip = !fieldInfo.required && (isRelational || fieldInfo.type === 'selection');
```

Comment:
Well, I meant that in the case where we want to use const + let in a file that has already var, if you have to add a lot of code, maybe you can rewrite all the var, but here, you only add 5 lines. So use var instead in this case.
Sorry for the confusion.
The idea is to avoid mixing writing conventions.